src/app/containers/MediaPlayer/index.jsx

```diff
@@ -48,27 +50,35 @@ const MediaPlayerContainer = ({ blocks, placeholder }) => {
     return null; // this should be the holding image with an error overlay
   }
 
+  const showPlaceholder = type === 'articles' || embedOverrides.showPlaceholder;
   const placeholderSrc = getPlaceholderSrc(imageUrl);
   const embedSource = embedUrl({
-    requestUrl: `${id}/${versionId}/${lang}`,
-    type: 'articles',
+    requestUrl: `${embedOverrides.id || id}/${versionId}/${lang}`,
+    type,
     isAmp,
     origin,
   });
 
+  const Wrapper = embedOverrides.wrapper || GridItemConstrainedMedium;
```

Comment:
Can you move this below `type` so all the `embedOverrides` stuff is together please?